java/runtime/src/main/java/org/ray/runtime/object/ObjectStore.java

```diff
@@ -29,15 +29,15 @@ public ObjectStore(WorkerContext workerContext) {
    * @param obj The ray object.
    * @return Generated ID of the object.
    */
-  public abstract ObjectId putRaw(NativeRayObject obj);
+  public abstract ObjectId putRaw(NativeRayObject obj, List<ObjectId> innerObjectIds);
```

Comment:
The `NativeRayObject` should contain the `innerObjectIds` so you don't need to add a parameter here.

Reply:
Yes.